### DISCUSSION TITLE
Creating Git repo from template fails

### DISCUSSION THREAD

**Discussion by bdnugget** [vercel/community#4468] [Help]
### Summary

I am not able to get a project working from a template. It initializes a git repo on my Github, but it stays empty. On the Vercel side it says: An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.
I have tried this multiple times with the nuxtjs/alpine blog theme, also tried it again after 5 hours. Tried it with a bare Nuxt template and with a Next template, all fail in the same step and give me an empty git repo.

I also tried to change my default branch on Github for new repos to main instead of master as the templates all use main but this made no difference either.

![image](https://github.com/vercel/community/assets/49720243/b516b617-25ab-4a69-8937-21c80789e922)

### Steps to Reproduce

Browse templates and pick a Nuxt.js or Next.js one
Click deploy
Give the repo a name and deploy
-> fails at the "Preparing Git Repository" stage, but does make an empty repo on Github.

**Comment by HEIN390** [comment]
I have exactly the same issue but with other templates

**Comment by amyegan** [comment]
An error at that phase of project creation suggests that there is a problem connecting to your GitHub account. You can check Vercel's permissions on your GitHub account by going to Settings > Applications and click "Configure" next to the Vercel app that should be listed there. If you go into your Vercel settings, you can completely reset the connection by removing and re-adding it.

Please check your GitHub connection and let me know if you're still having trouble with this 🙏

**Reply by Grisoly** [reply]
Hello @amyegan , 
I encounter the same issue and did the steps to reset the connection but it delivers the same error message.

**Reply by bdnugget** [reply]
To be clear: I can just manually clone or fork those repos and have Vercel deploy it just fine. It can connect to my Github just fine.
It goes wrong in the flow of selecting a template from the template showcase and having that one deploy automatically. The repo is created on Github (so there should be write access) but it just stays empty and shows the something went wrong error on the Vercel side.

**Reply by Grisoly** [reply]
Thanks for sharing @bdnugget, I will clone/fork them manually for now. ⏳️

**Comment by swarnava** [comment]
Do you  get any error in browser console? Can you send us a screencast of the reproductiob which we can look into?

**Reply by bdnugget** [reply]
I tried again just now and the problem seems to have automagically resolved itself. I successfully managed to deploy a Nuxt.JS project from the template and have it appear in my GitHub. So this ticket can be closed :)
